scripts/prep/global_ens/exevs_global_ens_wave_grid2obs_prep.sh

```diff
@@ -111,10 +115,12 @@ for ihour in 00 06 12 18 ; do
             chmod 640 $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc
             chgrp rstprod $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc
             if [ $SENDCOM = YES ]; then
-                cpreq -v $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/.
-                chmod 640 ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/gdas.${INITDATE}${ihour}.nc
-                chgrp rstprod ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/gdas.${INITDATE}${ihour}.nc
+                if [ -s $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc ]; then
+                    cp -v $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/.
+                fi
             fi
+            chmod 640 ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/gdas.${INITDATE}${ihour}.nc
+            chgrp rstprod ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/gdas.${INITDATE}${ihour}.nc
```

Comment:
I think lines 122 and 123 need to be placed back inside the if test. Right now the chmod and chgrp will throw system errors if SENDCOM=NO since the data file will not have been copied to com. I think you need this:
```
if [ $SENDCOM = YES ]; then
  if [ -s $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc ]; then
    cp -v $DATA/ncfiles/gdas.${INITDATE}${ihour}.nc ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/.
    chmod 640 ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/gdas.${INITDATE}${ihour}.nc
    chgrp rstprod ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}/gdas.${INITDATE}${ihour}.nc
  fi
fi
```

Reply:
I pushed that edit to the PR. Please see below.